src/Material/Lists.elm

```diff
@@ -0,0 +1,194 @@
+module Material.Lists
+    exposing
+        ( ul
+        , li
+        , twoLine
+        , threeLine
+        , itemPrimaryContent
+        , itemAvatar
+        , itemIcon
+        , itemSecondaryContent
+        , itemSecondaryInfo
+        , itemSecondaryAction
+        , itemTextBody
+        , itemSubTitle
+        )
+
+{-| From the [Material Design Lite documentation](http://www.getmdl.io/components/index.html#lists-section):
+
+> Lists present multiple line items vertically as a single continuous element.
+> Refer the Material Design Spec to know more about the content options.
+
+See also the
+[Material Design Specification]([https://material.google.com/components/lists.html).
+
+Refer to [this site](http://debois.github.io/elm-mdl#/lists)
+for a live demo.
+
+Example use:
```

Comment:
Either give a proper example, or just refer to the demo. I'm kind of prefer the the demo having a good example including code.